### PR TITLE
Update README.textile

### PR DESCRIPTION
**Pull request by olvap**
This fix the problem on ubuntu

### CODE REVIEW

File: README.textile [hunk 1]
```diff
@@ -69,6 +69,10 @@ the problem. Also see "this FAQ":https://web.archive.org/web/20140512193354/http
 
 On OS X with Homebrew, try [re]installing pkg-config:
 
+On Linux(you are awesome), try
```
Comment (linduxed) [inline README.textile]: I think we should avoid comments suggesting some sort of favoritism. This is a library that's open for use on all the supported platforms. I suggest you remove the parentheses and the contents within them.

Additionally, this isn't Linux-specific, but Ubuntu-specific, so this shouldn be reflected in the line. I also wonder if this is the case for all Debian-derivative distributions. Could you check in a virtual machine if the same happens for Debian Jessie, for instance?

File: README.textile [hunk 2]
```diff
@@ -69,6 +69,10 @@ the problem. Also see "this FAQ":https://web.archive.org/web/20140512193354/http
...
+
+bc. sudo apt-get install libmagickwand-dev
+
```
Comment (linduxed) [inline README.textile]: All of these changes have been put between the introductory line about OS X and the actual instructions for OS X, so your additions are squeezed in between. Either put it before the OS X instructions or after.

**Comment by linduxed** [comment]
I don't currently have a virtual machine to try to reproduce these instructions, but I'll set one up tomorrow (unless one of the others in the rmagick team has already tested this by then).

File: README.textile (hunk 2 above)
Comment (olvap) [inline README.textile]: OK, thanks for the advise. The line solved my problem on Ubuntu 14.

**Comment by vassilevsky** [comment]
Yeah I'm wondering how long we must live to see a hosted CI service that offers CentOS or Debian, not just Ubuntu machines.

**Comment by lmhsjackson** [comment]
Very helpful, thanks!

**Comment by vassilevsky** [comment]
Dear Oldani, thank you for this addition to the documentation. I have added it myself, in the `develop` branch. This pull request is now invalid because it targets `master` but we will be merging and committing all changes to `develop` between releases so I will close it. Please send other pull requests if you find something to improve. Here's a a small guide: 

https://github.com/rmagick/rmagick/blob/develop/CONTRIBUTING.md

**Comment by olvap** [comment]
Thanks. if I found something I will commit it.

**Pull request closed without merging**